lib/fluent/system_config.rb

```diff
@@ -58,6 +59,7 @@ class SystemConfig
     config_param :dir_permission, default: nil do |v|
       v.to_i(8)
     end
+    config_param :config_include_dir, default: ENV["FLUENT_CONF_DIR"] || '/etc/fluent/conf.d'
```

Comment:
It might be better to declare this as a constant in the https://github.com/fluent/fluentd/blob/510b891de1fb8a81065c8016bc5ac4f0eece4d63/lib/fluent/env.rb#L23-L26

Reply:
Thanks, it is reasonable.